### PR TITLE
if extension missing templates/project/manifest.rb, use discover :all

### PR DESCRIPTION
**Pull request by tka**

### CODE REVIEW

File: features/step_definitions/extension_steps.rb [hunk 1]
```diff
@@ -18,6 +18,13 @@
   end
 end
 
+Given /^I have a fake extension without manifest file at (.*)$/ do |directory|
+  directory.gsub!('~', ENV["HOME"]) if directory.include?('~/')
```
Comment (scottdavis) [inline features/step_definitions/extension_steps.rb:22]: I donno if i like this in a test case using my home directory to run tests in is kinda making me queezy we probaly should mock this in tests /cc @chriseppstein

Comment (chriseppstein) [inline features/step_definitions/extension_steps.rb:22]: ya that's probably right.

**Comment by scottdavis** [comment]
going t cherry-pick these changes in later im scared to pull this in with tests accessing the home directory without mocking

**Comment by chriseppstein** [comment]
I merged this in without the test.

**Pull request closed without merging**